back-office/dashboard.md

```diff
@@ -0,0 +1,12 @@
+# **SPECIFICATIONS - DASHBOARD PAGE**
+
+
+_As a user, I want to know when my PHP version is no longer supported._
+
+Since 1.7.8, a yellow warning notification is displayed on the top of the Dashboard page when the PHP version detected is no longer supported, cf. issue #[20862](https://github.com/PrestaShop/PrestaShop/issues/20862).
+
+It says `You are using a PHP version that is not supported and this can affect your store. Please check PHP requirements and change for the suitable PHP version.` in `Admin.Notifications.Warning`.
+
+Users should be able to dismiss this warning notification when it pops up, but it will keep on appearing at each session until the PHP version is changed.
```

Comment:
What about multistore?

When I dismiss the warning in shop 1, does it also dismiss it for shop 2 or do we consider that this is 2 different shops ?

By users I suppose you mean BackOffice users, right?

Reply:
Good question! I suggest considering the shops as different to make sure the (BO, you suppose right, @matks) users get the message. In any case, the warning is not too intrusive and displayed only when users reach the Dashboard page. What would be your recommendation, @marionf?

Reply:
To be consistent with other informations messages (described in the specs [here](https://github.com/PrestaShop/prestashop-specs/blob/master/back-office/multistoregeneralspecs.md)), I recommand to close the message only for the selected context and to not redisplay it if the user refreshes or returns on the page later

Reply:
So if we go back to @matks example: if the notification is dismissed for shop 1, it does not automatically dismiss it for shop 2, right?

Reply:
Exactly !

Reply:
Many thanks, I'll use the same sentence as in your specs.